docs/man_pages/general/usage-reporting.md

```diff
@@ -1,16 +1,16 @@
-feature-usage-tracking
+usage-reporting
 ==========
 
 Usage | Synopsis
 ------|-------
-General | `$ appbuilder feature-usage-tracking [<Command>]`
+General | `$ appbuilder usage-reporting [<Command>]`
 
-Configures anonymous usage statistics tracking for the Telerik AppBuilder CLI.
+Configures anonymous usage reporting for the Telerik AppBuilder CLI.
 
 All data gathered is used strictly for improving the product and will never be used to identify or contact you.
 
 ### Attributes
-`<Command>` extends the `feature-usage-tracking` command. You can set the following values for this attribute.
+`<Command>` extends the `usage-reporting` command. You can set the following values for this attribute.
 * `status` - Shows the current configuration for anonymous usage tracking for the Telerik AppBuilder CLI.
```

Comment:
the dreaded tracking is here. Please, change it to reporting for status, disable and enable.